os/include/tinyara/kernel_test_drv.h

```diff
@@ -30,14 +30,14 @@
 #include <tinyara/config.h>
 #include <tinyara/fs/ioctl.h>
 
-#ifdef CONFIG_DRIVERS_KERNEL_TEST
+#ifdef CONFIG_OS_KERNEL_TEST
```

Comment:
We will also need to rename this header file as os_api_test_drv.h

Reply:
done